src/scripts/math_controller.js

```diff
@@ -0,0 +1,22 @@
+import { Controller } from 'stimulus-repo/packages/stimulus'
+
+export class MathController extends Controller {
```

Comment:
I broke out the practice and test code into their own controllers. The code had the prefix "text" on a bunch of variables; that's often a code smell indicating that it should be broken out on its own.

With all of that code moved out, this code is mostly just handling the tabs and calling each of the other controllers. It's still sort of mission control and might take on more functionality. Maybe picking between multiplication and division?